chad_stride.c

```diff
@@ -221,7 +221,9 @@ void usage(const int exit_code) {
             "-a SLEEP_TIME: Adjust sleep timer for chad\n"
             "-c COLOR: Chad strolls by in the color of your choice (r|g|b|m|c|y)\n"
 	    "-m MESSAGE: Display message under chad [capped at 50 characters]\n"
+      "-R RAINBOW: Have Chad stroll with a rainbow effect\n"
```

Comment:
One thing otherwise this is perf, can you get rid of the RAINBOW after -R. The all caps var name after the flag is used to signify there is another input value like `./cs -a 3000` or something. So if you just made it `-R: Have Chad...` that would be great

Reply:
Okay, I made the change. Let me know if I have to create another pull request.